shell/common/rasterizer.cc

```diff
@@ -68,6 +68,17 @@ void Rasterizer::Teardown() {
   last_layer_tree_.reset();
 }
 
+void Rasterizer::PurgeCaches() const {
+  if (!surface_) {
+    return;
```

Comment:
Please add log statements for short-circuit returns on debug builds.

Reply:
Done